client/domain/QuestionObjectFactorySpec.js

```diff
@@ -78,6 +85,10 @@ describe('QuestionObjectFactory', function() {
       expect(question.isLastTask(2)).toBe(false);
       expect(question.isLastTask(-1)).toBe(false);
       expect(question.isLastTask(1)).toBe(true);
+      expect(question.isLastTask(question.getTasks.length -
```

Comment:
I think I commented on this before, but I still don't understand the point of the "Z - (Z - 1)" construct...